### ISSUE TITLE
Bread input file with optional validation

### ISSUE THREAD

**Issue by howdu** [thedevdojo/voyager#5422]
### Laravel version

8

### PHP version

7.4

### Voyager version

1.4

### Database

Mysql

### Description

Creating a 'file' field with laravel optional validation is not possible due to the value being `[]` instead of null.

Problem fixed by removing this line code:
https://github.com/the-control-group/voyager/blob/110229618c035600fc55c2104bd629701d96c283/src/Http/Controllers/Controller.php#L109

        
### Steps to reproduce

E.g adding field details:
{
    "validation": {
        "rule": [
            "nullable",
            "max:10240",
            "mimes:pdf,txt,doc,docx"
        ]
    }
}

### Expected behavior

Null not []

**Comment by karam-mustafa** [comment]
@howdu  Do you mean that you want the file to be null?  not an encoded JSON?

**Comment by howdu** [comment]
Yes I wanted the validation of the file to be optional. E.g if there's no file passed it will be null and json coding not needed.
